nginx.conf

```diff
@@ -0,0 +1,18 @@
+events {
+  worker_connections 768;
+}
+http {
+  upstream dokomo {
+    server webapp:8888;
```

Comment:
So because docker is setting up the link to webapp under the nginx section in docker-compose.yml, nginx knows the web server by hostname webapp? Nifty.